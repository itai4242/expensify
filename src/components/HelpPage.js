import React from 'react';

const HelpPage = () =>(
    <div>
        help me!
    </div>
)
export default HelpPage;