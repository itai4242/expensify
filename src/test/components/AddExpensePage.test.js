import React from 'react'
import {shallow} from 'enzyme';
import {ADDExpensePage} from '../../components/ADDExpensePage'
import expenses from '../fixtures/expenses'

let addExpense, history, wrapper;
beforeEach( () => {
    addExpense= jest.fn();
    history = { push: jest.fn() };
    wrapper =  shallow(<ADDExpensePage addExpense={addExpense} history = {history} exp />);
})

test ('should render ADDExpensePage correctly',() => {
    expect(wrapper).toMatchSnapshot();
})

test ('should handle onsubmit',() => {
    wrapper.find('ExpenseForm').prop('onSubmit')(expenses[1]);
    expect(history.push).toHaveBeenLastCalledWith('/');
    expect(addExpense).toHaveBeenLastCalledWith(expenses[1])
})