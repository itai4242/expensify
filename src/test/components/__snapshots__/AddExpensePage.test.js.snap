// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ADDExpensePage correctly 1`] = `
<div>
  <h1>
    Add Eepense
  </h1>
  <ExpenseForm
    onSubmit={[Function]}
  />
</div>
`;
